src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -62,7 +64,27 @@ const CharacterForm = ({ onClose }) => {
     return params[0] === formModeEnum.edit
   }, [params])
 
-  const character = useSelector(characterSelectors.character(params[1]))
+  const characterCacheId = client.cache.identify({
+    __typename: "Person",
+    id: params[1],
+  })
+
+  const {
+    error: getACharacterError,
+    loading: getACharacterLoading,
```

Comment:
we prefix booleans with words like "is", "has", "should" etc, so we can tell by the name they are booleans